### PR TITLE
CORENET-7554: Add disruptive conformance suite for OTE

### PR DESCRIPTION
**Pull request by anuragthehatter**
## Summary
- Add `ovn-kubernetes/conformance/disruptive` suite with `Parallelism=1` and `ClusterStabilityDisruptive`
- Tests labeled with `g.Label("Disruptive")` are routed to this suite
- The `ClusterStabilityDisruptive` flag tells monitors to expect node disruptions (NotReady, Unreachable), suppressing false failures from intentionally disruptive tests like node reboots

Depends-On: https://github.com/openshift/release/pull/85072

## Test plan
- [ ] Verify suite is discoverable via `openshift-tests list-suites`
- [ ] Verify disruptive-labeled tests are routed to this suite
- [ ] Verify monitor tests pass when running disruptive tests

🤖 Generated with [Claude Code](https://claude.com/claude-code)

### CODE REVIEW

**Comment by anuragthehatter** [comment]
@coderabbitai resolve

File: openshift/cmd/ovn-kubernetes-tests-ext/main.go [hunk 1]
```diff
@@ -95,6 +95,13 @@ func main() {
 		Qualifiers: []string{`!labels.exists(l, l == "Serial")`},
 	})
 
+	ovnTestsExtension.AddSuite(extension.Suite{
+		Name:             "ovn-kubernetes/conformance/disruptive",
+		Qualifiers:       []string{`labels.exists(l, l == "Disruptive")`},
+		Parallelism:      1,
+		ClusterStability: extension.ClusterStabilityDisruptive,
+	})
```
[the remark this answers is not in the document]
Comment (anuragthehatter) [reply openshift/cmd/ovn-kubernetes-tests-ext/main.go:103]: Addressed

File: openshift/cmd/ovn-kubernetes-tests-ext/main.go [hunk 2]
```diff
@@ -95,6 +95,13 @@ func main() {
 		Qualifiers: []string{`!labels.exists(l, l == "Serial")`},
 	})
 
+	ovnTestsExtension.AddSuite(extension.Suite{
+		Name:             "ovn-kubernetes/conformance/disruptive",
```
[the remark this answers is not in the document]
Comment (anuragthehatter) [reply openshift/cmd/ovn-kubernetes-tests-ext/main.go:99]: Addressed

**Comment by anuragthehatter** [comment]
This is blocking https://github.com/openshift/ovn-kubernetes/pull/3446 merge 
@tssurya if you can help merging it. It has lgtm'ed by @jluhrsen

**Comment by anuragthehatter** [comment]
> This is blocking #3446 merge @tssurya if you can help merging it. It has lgtm'ed by @jluhrsen

@tssurya ^ Just a gentle reminder

**Comment by tssurya** [comment]
> > This is blocking #3446 merge @tssurya if you can help merging it. It has lgtm'ed by @jluhrsen
> 
> @tssurya ^ Just a gentle reminder

same answer as here: https://redhat-internal.slack.com/archives/C0B3T4NEQSD/p1790078233299639?thread_ts=1789607397.120309&cid=C0B3T4NEQSD

"I am aware of this, sorry for the delay." (Please note reviews are getting prioritized as per team priority, example EIP OTE has higher priority than corner case day2 masqueradeIP change disruption tests landing, that's the rationale behind review delay but its on my list for sure, will get to it)

**Comment by anuragthehatter** [comment]
> > > This is blocking #3446 merge @tssurya if you can help merging it. It has lgtm'ed by @jluhrsen
> > 
> > 
> > @tssurya ^ Just a gentle reminder
> 
> same answer as here: https://redhat-internal.slack.com/archives/C0B3T4NEQSD/p1790078233299639?thread_ts=1789607397.120309&cid=C0B3T4NEQSD
> 
> "I am aware of this, sorry for the delay." (Please note reviews are getting prioritized as per team priority, example EIP OTE has higher priority than corner case day2 masqueradeIP change disruption tests landing, that's the rationale behind review delay but its on my list for sure, will get to it)

Thanks. Just want to make sure its not just for masq day 2 cases but for all open OTE PRs where disrutptive tags will be added/refactored and re-rehearsals needs to be validated on all open PRs. This is critical change in ovnk main.go IMO and can be leveraged in EIP OTE work as well if any disruptive cases are present there.

**Comment by tssurya** [comment]
> > > > This is blocking #3446 merge @tssurya if you can help merging it. It has lgtm'ed by @jluhrsen
> > > 
> > > 
> > > @tssurya ^ Just a gentle reminder
> > 
> > 
> > same answer as here: https://redhat-internal.slack.com/archives/C0B3T4NEQSD/p1790078233299639?thread_ts=1789607397.120309&cid=C0B3T4NEQSD
> > "I am aware of this, sorry for the delay." (Please note reviews are getting prioritized as per team priority, example EIP OTE has higher priority than corner case day2 masqueradeIP change disruption tests landing, that's the rationale behind review delay but its on my list for sure, will get to it)
> 
> Thanks. Just want to make sure its not just for masq day 2 cases but for all open OTE PRs where disrutptive tags will be added/refactored and re-rehearsals needs to be validated on all open PRs. This is critical change in ovnk main.go IMO and can be leveraged in EIP OTE work as well if any disruptive cases are present there.

I understand that, that's the point I am trying to make :D let's get the functional OTE pieces in first, i.e the basic e2es are missing :D disruption suite follows that in terms of priority (I was trying to explain my review backlog queue) - that I understand I nee to review it, will surely get to it :)
